docs/source/getting_started/benchmarks/cifar10_benchmark.py

```diff
@@ -339,23 +401,28 @@ def configure_optimizers(self):
         scheduler = torch.optim.lr_scheduler.CosineAnnealingLR(optim, max_epochs)
         return [optim], [scheduler]
 
-model_names = ['MoCo_128', 'SimCLR_128', 'SimSiam_128', 'BYOL_128',
-               'MoCo_512', 'SimCLR_512', 'SimSiam_512', 'BYOL_512']
-models = [MocoModel, SimCLRModel, SimSiamModel, BYOLModel]
+model_names = ['MoCo_128', 'SimCLR_128', 'SimSiam_128', 'BarlowTwinsModel_128', 'BYOL_128',
+               'MoCo_512', 'SimCLR_512', 'SimSiam_512', 'BarlowTwinsModel_512', 'BYOL_512']
```

Comment:
Now you didn't add SwaV?

Reply:
Thanks for pointing that out. I pushed the PR too early